Src/FluentAssertions/Formatting/StringValueFormatter.cs

```diff
@@ -23,7 +23,9 @@ public string Format(object value, FormattingContext context, FormatChild format
             string prefix = context.UseLineBreaks ? Environment.NewLine : "";
             string escapedString = value.ToString();
 
-            return prefix + "\"" + escapedString + "\"";
+            return escapedString == Environment.NewLine
```

Comment:
Instead of wrapping all new line inside a `AlreadyFormattedString` I have decided to update the string formatter to avoid wrapping them (this makes more sense IMO).